whatnext/__init__.py

```diff
@@ -1,14 +1,15 @@
 """
 This module contains the initial Flask configuration for the 'What next' application
 """
+import os
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-import os
+from whatnext import routes
 
 app = Flask(__name__)
 db_file = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', 'data','whatnext.db'))
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 app.config['SQLALCHEMY_DATABASE_URI'] = "sqlite:///%s"%db_file
 db = SQLAlchemy(app)
 
-from whatnext import routes
```

Comment:
I think moving this up will cause a circular dependency and the app is not going to start. I tried starting and it didn't work. On a quick search, found that its a circular dependency - routes depend on app, app depends on route.

https://stackoverflow.com/questions/9252543/importerror-cannot-import-name-x

Reply:
I moved the import back down and the app started fine.

Reply:
Thanks, When I run the pylint it complaint about import order. So I changed it. I moved it back now